### ISSUE TITLE
Fix pinch zoom on iOS 13

### ISSUE THREAD

**Issue by line72** [line72/montclair#33]
iOS 13 doesn't appear to respect the existing view port, and allows pinch-zoom on the title and route list, breaking zooming on the map.

It appears that adding the following should resolve the issue:
```
<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,viewport-fit=cover,user-scalable=no,shrink-to-fit=no">
<meta name="HandheldFriendly" content="true">
```

**Comment by line72** [comment]
I ended up not using the viewport-fit=cover, since on iOS, it covered the top bar.

**Comment by line72** [comment]
Merged in master in 0fca89e26db1e439f9704ab00de35c537f398852

**Comment by line72** [comment]
https://stackoverflow.com/a/51710577/4122137
